src/api/app/components/patchinfo_component.html.haml

```diff
@@ -36,7 +36,7 @@
     - if @patchinfo['releasetarget'].present?
       %h6 Targeted for release in the following projects
       %ul
-        - @patchinfo['releasetarget'].each do |releasetarget|
+        - [@patchinfo['releasetarget']].flatten.each do |releasetarget|
```

Comment:
I would move all those `[@something].flatten` into the component class, expose it with a method or an instance variable

Reply:
@danidoni  please have another look